typescript/custom-resource/my-custom-resource.ts

```diff
@@ -22,8 +22,7 @@ export class MyCustomResource extends cdk.Construct {
         uuid: 'f7d4f730-4ee1-11e8-9c2d-fa7ae01bbebc',
         code: new lambda.InlineCode(fs.readFileSync('custom-resource-handler.py', { encoding: 'utf-8' })),
         handler: 'index.main',
-        timeout: 300,
```

Comment:
Replace with:
``` typescript
timeout: cdk.Duration.seconds(300),
```

Reply:
@NGL321 @tom98784 The default configuration for the `FunctionProp.timeout` is 3 seconds. Is the example calls out for 300 seconds? If so we can change it, otherwise I would argue that it is redundant.

Reply:
3c14df84b432527c31e4e4a05a6fcba73e59d0ca